api/PresentationConnection.json

```diff
@@ -194,16 +194,15 @@
           }
         }
       },
-      "id": {
+      "close_event": {
         "__compat": {
-          "mdn_url": "https://developer.mozilla.org/docs/Web/API/PresentationConnection/id",
-          "spec_url": "https://w3c.github.io/presentation-api/#dom-presentationconnection-id",
+          "description": "<code>close</code> event",
           "support": {
             "chrome": {
-              "version_added": "47"
+              "version_added": "50"
```

Comment:
This doesn't match the parent interface and made me suspect that on* properties were missing while the events were actually supported, but per https://source.chromium.org/chromium/chromium/src/+/5458a1b5f82ef7a0d01fca7c7352cd7fe7ec5d9a the events and the properties were added together. `PresentationConnectionCloseEvent` is also marked as Chrome 50.